src/main/java/com/codesquad/rare/domain/post/PostController.java

```diff
@@ -28,4 +32,19 @@
   public ApiResult<Post> findByPostId(@PathVariable(value = "id") Long id) {
     return OK(postService.findById(id));
   }
+
+  //Post 를 생성합니다.
+  //method 명은 save, create? 뭐가 좋을까요?
+  @PostMapping
+  public ApiResult save(@RequestBody PostRequestDto postRequestDto) {
+    postService.save(postRequestDto);
+    return OK("포스트 생성 성공");
+  }
+
+  //Post 를 삭제합니다.
+  @DeleteMapping("/{id}")
+  public ApiResult delete(@PathVariable("id") Long postId) {
+    postService.delete(postId);
+    return OK("포스트 삭제 성공");
```

Comment:
포스트가 삭제 되었을 때는
```suggestion
return OK(true);
```
이처럼 리턴 하는게 깔끔하지 않을까요?